eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/NewEpoch.hs

```diff
@@ -151,12 +151,17 @@ newEpochTransition = do
       let updateRewards ru'@(RewardUpdate dt dr rs_ df _) = do
             let totRs = sumRewards (esPrevPp es) rs_
             Val.isZero (dt <> (dr <> toDeltaCoin totRs <> df)) ?! CorruptRewardUpdate ru'
-            let (es', regRU) = applyRUpd' ru' es
-            tellEvent $ RewardEvent e regRU
+            let (es', _regRU) = applyRUpd' ru' es
+            -- tellEvent $ RewardEvent (RupdEvent e regRU)
```

Comment:
we can remove this comment now, leader and member rewards are already sent by this time

Reply:
done